index.js

```diff
@@ -39,11 +41,13 @@ function rollupify(filename, opts) {
       }
 
       return rollup.rollup(Object.assign(config, {
+        cache: cache,
         entry: tmpfile,
         sourceMap: doSourceMap ? 'inline' : false
       }))
     }).then(function (bundle) {
       var generated = bundle.generate({format: 'cjs'});
+      cache = bundle;
```

Comment:
Ah interesting, `cache` doesn't seem documented anywhere, but it does seem to do the correct internal caching.